components/sidebar/add_channel_dropdown.tsx

```diff
@@ -182,7 +188,25 @@ const AddChannelDropdown = ({
                 id='AddChannelDropdown'
                 ariaLabel={intl.formatMessage({id: 'sidebar_left.add_channel_dropdown.dropdownAriaLabel', defaultMessage: 'Add Channel Dropdown'})}
             >
-                {renderDropdownItems()}
+                <Menu.Group>
+                    {renderDropdownItems.map((item, index) => (
+                        <Menu.Group divider={<li className={item.divider}/>}> {/*eslint-disable-line*/}
```

Comment:
Why can't divider go on a new line?

Reply:
Does this remove the previous groupings we had of the menu items?

Reply:
No, the groupings are same.